### PR TITLE
CLOUDP-73722: mongocli ops-manager admin backup blockstore(s) delete

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-73722

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

`./bin/mongocli om admin backups blockstore delete test -P ops
? Are you sure you want to delete: test Yes
Blockstore configuration 'test' deleted`

### CODE REVIEW

**Comment by andreaangiolillo** [comment]
Waiting for https://github.com/mongodb/mongocli/pull/475

File: internal/cli/opsmanager/admin/backup/blockstore/delete.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+func (opts *DeleteOpts) init() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *DeleteOpts) Run() error {
+	return opts.Delete(opts.store.DeleteBlockstore)
```
Comment (gssbzn) [inline internal/cli/opsmanager/admin/backup/blockstore/delete.go:38]: [q] does this work? doesn't this method assume a project ID? which you may not have?

**Review by gssbzn — APPROVED** [review]
Left a question but if you tested this and work I guess that method is more magical than I remember

File: internal/cli/opsmanager/admin/backup/blockstore/delete.go (hunk 1 above)
Comment (andreaangiolillo) [reply internal/cli/opsmanager/admin/backup/blockstore/delete.go:38]: This endpoint does not need the projectID https://docs.opsmanager.mongodb.com/current/reference/api/admin/backup/snapshot/mongoConfigs/delete-one-blockstore-configuration/.

Comment (gssbzn) [reply internal/cli/opsmanager/admin/backup/blockstore/delete.go:38]: correct, so my question is will `opts.Delete` work, as I thought it handled the project ID, unless it doesn't

Comment (andreaangiolillo) [reply internal/cli/opsmanager/admin/backup/blockstore/delete.go:38]: Yes, I tested it and It works. I don't think the ProjectID is handled by opts.Delete (I quick looked at the code)

**Pull request merged**